_new_content_templates/configuration-option-setting.md

```diff
@@ -44,3 +44,10 @@ tinymce.init({
   <configuration_option>: false
 });
 ```
+
+{% comment %} Remove if not applicable {% endcomment %}
+### Limitation of the `<configuration_option>` option
+
+The `<configuration_option>` option has the following limitations.
+
+<List the things we can't or wont fix, as well as complicated complications and risks>
```

Comment:
This appears to be a TODO?

Reply:
Template page for creating new documentation content, not a real page :smile: